pkg/pss/evaluate.go

```diff
@@ -84,6 +155,65 @@ func exemptKyvernoExclusion(defaultCheckResults, excludeCheckResults []pssutils.
 	return newDefaultCheckResults
 }
 
+func extractBadValues(excludeFieldErr *field.Error) []string {
+	var excludeBadValues []string
+
+	switch excludeFieldErr.BadValue.(type) {
+	case string:
+		badValue := excludeFieldErr.BadValue.(string)
+		if badValue == "" {
+			break
+		}
+		excludeBadValues = append(excludeBadValues, badValue)
+	case bool:
+		excludeBadValues = append(excludeBadValues, strconv.FormatBool(excludeFieldErr.BadValue.(bool)))
+	case int:
+		excludeBadValues = append(excludeBadValues, strconv.Itoa(excludeFieldErr.BadValue.(int)))
+	case []string:
+		excludeBadValues = append(excludeBadValues, excludeFieldErr.BadValue.([]string)...)
+	default:
```

Comment:
I don't recommend having an empty default clause. We either remove it or return an error.